### PR TITLE
Update README and License

### PR DESCRIPTION
**Pull request by djalova**
I updated the README and setup.py so the new README should be displayed on PyPi. I also added IBM to the license file and deleted an unused file.

### CODE REVIEW

**Comment by djalova** [comment]
Adresses #11

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+# Model Asset Exchange Framework
+ Python package that contains common code shared across all MAX models.
+
+## Dependencies
+* [flask-restplus](https://pypi.org/project/flask-restplus/0.11.0/)
+* [flask-cors](https://pypi.org/project/Flask-Cors/)
+
+## Installation
+The package can be installed with pip. However, this is not necessary as the
+Docker images of all MAX models will include `maxfw` as a package.
```
Comment (bdwyer2) [inline README.md]: Maybe change this to something like:
```
The package can be installed with pip. However, this is not necessary as each MAX model will get the `maxfw` package via the `MAX-Base` image`.

Comment (djalova) [reply README.md]: Thanks for the suggestion, I think is a better way to put it.

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019, IBM
```
Comment (ptitzler) [inline LICENSE]: Copyright 2019 IBM

Comment (MLnick) [reply LICENSE]: Is it not supposed to be IBM Corporation?

**Pull request merged**